docs/metrics-docs/metrics-api.en-GB.md

```diff
@@ -1,29 +1,369 @@
+      "source": "OAPEN",
+      "type": "downloads",
+      "version": "v1",
+      "value": 209
+    },
+    {
+      "measure_uri": "https://metrics.operas-eu.org/up-logs/downloads/v1",
+      "namespace": "https://metrics.operas-eu.org",
+      "source": "Ubiquity Press",
+      "type": "downloads",
+      "version": "v1",
+      "value": 313
+    },
+    ...
+  ]
+}
+```
+
+> Request Example with start date and end date
+
+`http://localhost:8080/events?start_date=2020-09-18&end_date=2024-08-18`
+
+> Response Example with start date and end date
+
+```json
+{
+    "status": "ok",
+    "code": 200,
+    "count": 2,
+    "data": [
+        {
+            "event_id": "fa5fcb71-78ca-4183-a44b-d07030bc5e80",
+            "work_uri": "info:doi:10.5334/jors.122",
+            "measure_uri": "https://metrics.operas-eu.org/crossref/citations/v1",
+            "timestamp": "2022-08-29T01:00:00+0100",
+            "value": 1,
+            "event_uri": "10.5194/gmd-15-6399-2022",
+            "country_uri": null,
+            "uploader_uri": "acct:tech@ubiquitypress.com"
+        },
+        {
+            "event_id": "da55c8ab-12df-4885-8b92-edff51bc23d0",
+            "work_uri": "info:doi:10.5334/jors.148",
+            "measure_uri": "https://metrics.operas-eu.org/crossref/citations/v1",
+            "timestamp": "2022-11-08T00:00:00+0000",
+            "value": 1,
+            "event_uri": "10.5194/npg-30-13-2023",
+            "country_uri": null,
+            "uploader_uri": "acct:tech@ubiquitypress.com"
+        }
+    ]
+}
+```
+
+
+### POST `events`
+
+#### Params
+
+|Name|Location|Type|Required|Description|
+|---|---|---|---|---|
+|work_uri|body|string| yes |Uniform Resource Identifier (URI) that uniquely identifies a specific work or piece of content saved in the Event table.|
+|measure_uri|body|string| yes |A reference to a service or endpoint that provides metrics or measurements.|
+|timestamp|body|string| yes |Date in ISO 8601 format.|
+|value|body|integer| yes |Number of times this event entry occurred.|
+|event_uri|body|string| no |URI used for single-event metrics, to give more information about the event. The format is specific to the type of event.|
+|country_uri|body|string| no |URN (Uniform Resource Name) format specified by the ISO 3166 standard, which is used for country codes. Examples: `urn:iso:std:3166:-2:US` (United States), `urn:iso:std:3166:-2:PK` (Pakistan)|
+|uploader_uri|body|string| yes |URI to identify the user that uploaded the metrics event. This uses the acct URI scheme|
+
+> Request Example
+
+`http://localhost:8080/events`
+
+- Body Parameters
+
+```json
+{
+    "work_uri": "http://example.com/work/2",
+    "measure_uri": "http://example.com/measure/2",
+    "timestamp": "2023-10-01T12:00:00+0000",
+    "value": 100,
+    "event_uri": "http://example.com/event/2",
+    "country_uri": "http://example.com/country/2",
+    "uploader_uri": "http://example.com/uploader/2"
```

Comment:
This is not a valid `uploader_uri` value. It needs to follow the acct URI scheme.
Look at the example here: https://metrics-api.operas-eu.org/events?filter=work_uri:info:doi:10.5334/bcd&start_date=2024-10-25